### PR TITLE
Platform/pm 19/platform team file moves

### PR DESCRIPTION
**Pull request by MGibson1**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [x] Other
```

## Objective

Move files to Platform team owned directories.

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: libs/common/src/platform/services/cryptography/encrypt.service.implementation.ts [hunk 1]
```diff
@@ -187,7 +187,7 @@ export class EncryptServiceImplementation implements EncryptService {
    * Transform into new key for the old encrypt-then-mac scheme if required, otherwise return the current key unchanged
    * @param encThing The encrypted object (e.g. encString or encArrayBuffer) that you want to decrypt
    */
-  resolveLegacyKey(key: SymmetricCryptoKey, encThing: IEncrypted): SymmetricCryptoKey {
+  resolveLegacyKey(key: SymmetricCryptoKey, encThing: encrypted): SymmetricCryptoKey {
```
Comment (Hinton) [inline libs/common/src/platform/services/cryptography/encrypt.service.implementation.ts:190]:
```suggestion
  resolveLegacyKey(key: SymmetricCryptoKey, encThing: Encrypted): SymmetricCryptoKey {
```

**Review by Hinton — APPROVED** [review]
Gone through all the code and found a few things. Will need to look through what's moved as well.

**Comment by Hinton** [comment]
Suggestions:

* Make `apps/web/src/core` owned by platform and move back core services.
* libs/angular/src/services (except jslib-services) should be moved and owned by platform?
* libs/common/src/abstractions & services. I think we should move more services into platform.
I would like to keep the `core` module in `apps/web`, and mark it as owned by platform, I'm a bit unsure why some of the files are moved out, and some weren't.

**Comment by MGibson1** [comment]
mostly file moves were done according the the document I made up months and months ago. Given the size of this thing, I didn't want to go rogue _too_ much.

I agree there will definitely be more moves to the platform team over time. I'm OK doing them now. I had thought your 2nd and 3rd points while doing the move as well.

first point sounds good too, I'll get a commit in

**Review by Hinton — APPROVED** [review]
Gone through all the files and verified only imports are changed. LGTM.

**Pull request merged**